asammdf/gui/ui/file_widget.ui

```diff
@@ -882,7 +888,7 @@
   </layout>
  </widget>
  <resources>
-  <include location="resource.qrc"/>
+  <include location="../../../../../asammdf/asammdf/gui/ui/resource.qrc"/>
```

Comment:
This relative path works with the setup on your PC, but it will not work for others. Please change from relative part to ``<include location="resource.qrc"/>``